src/analyse/adapt-dependencies/adapt-dependencies.ts

```diff
@@ -9,18 +9,18 @@ import type { Module, PackageAfterDetermineVersion } from '../../types';
 import { logger } from '../../helpers/logger';
 
 import { getReleasablePackages } from '../../helpers/get-releasable-packages';
+import { missingEnvAttrError } from '../../helpers/validation-errors';
```

Comment:
You could implement this inside `helpers/logger`. There you already have the topic. Makes the API much simpler.

Reply:
I'd also name it `throwMissingAttrError`. Makes it more clear, that we throw an error inside.